tools/remove_dependencies.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+# Copyright 2019 The Vitess Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Remove tools and dependencies installed by "make tools"
+
+get_arch() {
+  uname -m
+}
+
+function fail() {
+	echo "ERROR: ${1}"
+	exit 1
+}
+
+BUILD_JAVA=${BUILD_JAVA:-1}
+BUILD_CONSUL=${BUILD_CONSUL:-1}
+VTROOT="$PWD/../"
+
+[[ "$(dirname "$0")" = "." ]] || fail "remove_dependencies.sh must be run from its current directory"
+
+uninstall_protoc() {
+    echo "Removing protoc..."
+    local dist="$1"
+
+    if [ -f $dist ]; then
+        unlink "$dist/bin/protoc"
+        rm "$VTROOT/bin/protoc"
+    fi
+    rm -rf $dist
+}
```

Comment:
It seems like the symlink on `protoc` is still present even after executing this function.

```
root@x:~/vitess# ls bin/
consul  etcd  etcdctl  protoc  toxiproxy-server  zksrv.sh

root@x:~/vitess# cd tools/

root@x:~/vitess/tools# ./remove_dependencies.sh 
## local system details...
## platform: Linux target:x86_64 OS: 
Removing protoc...
Removing zookeeper...
Removing etcd...
Removing consul...
Removing toxiproxy...

root@x:~/vitess/tools# ls ../bin/
protoc  zksrv.sh

root@x:~/vitess/tools# cd ..

root@x:~/vitess# ./bin/protoc
-bash: ./bin/protoc: No such file or directory

root@x:~/vitess# ls bin -l 
total 0
lrwxrwxrwx 1 root root 43 Oct 18 22:50 protoc -> /root/vitess/dist/vt-protoc-21.3/bin/protoc
lrwxrwxrwx 1 root root 33 Oct 18 22:52 zksrv.sh -> /root/vitess/go/vt/zkctl/zksrv.sh
```

Reply:
On another note, I think the symlink to `zksrv.sh` is fine to keep as it gets created by `build.env`